Test/Test2.js

```diff
@@ -0,0 +1,8 @@
+var array = [0, 1];
+
+var a = Math.random();
```

Comment:
better: 
```javascript
(function() return 0.5;)(); 
```
because than we always do the conditioned push, else it might be optimized to ```array[0, 1, 2]```